src/docker/compose_helpers.go

```diff
@@ -17,7 +17,7 @@ func CreateNewContainer(serviceName string, env []string, dockerComposeDir strin
 
 // KillAllContainers kills all the containers
 func KillAllContainers(dockerComposeDir string, logChannel chan string) (*execplus.CmdPlus, error) {
-	cmdPlus := execplus.NewCmdPlus("docker-compose", "down")
+	cmdPlus := execplus.NewCmdPlus("docker-compose", "down", "--remove-orphans")
```

Comment:
I'm good removing this. Was just hitting it locally and that seemed to be what it suggested. I created #441 to do something like it in our tests.